instrumentation/vertx-reactive-3.5/javaagent/build.gradle.kts

```diff
@@ -10,12 +11,21 @@ muzzle {
   }
 }
 
-//The first Vert.x version that uses rx-java 2
-val vertxVersion = "3.5.0"
```

Comment:
This still seems useful, considering in how many places below this value is used

Reply:
added it back